tests/scripts/all.sh

```diff
@@ -3229,9 +3225,8 @@ component_build_psa_accel_key_type_ecc_public_key() {
 
 # This should be renamed to test and updated once the accelerator HMAC code is in place and ready to test.
 component_build_psa_accel_alg_hmac() {
-    msg "build: full + MBEDTLS_PSA_CRYPTO_CONFIG + PSA_WANT_ALG_HMAC"
+    msg "build: full - MBEDTLS_USE_PSA_CRYPTO + PSA_WANT_ALG_HMAC"
```

Comment:
So that I understand:
Before we were using MBEDTLS_PSA_CRYPTO_CONFIG and now we say that we are removing it, but I don't see the unset command. I think the comment "- MBEDTLS_USE_PSA_CRYPTO" may need to be removed?

Reply:
Before, we were setting `MBEDTLS_PSA_CRYPTO_CONFIG` explicitly. Now that `full` sets it, this is unnecessary.

`MBEDTLS_USE_PSA_CRYPTO` is unrelated. It was and still is set by `full`, and this particular component unsets it because it wants it disabled.